digitalocean/baseapi.py

```diff
@@ -97,6 +97,32 @@ def __perform_request(self, url, type=GET, params=None):
 
         return requests_method(url, **kwargs)
 
+    def __deal_with_pagination(self, url, method, params, data):
+        """
+            Perform multiple calls in order to have a full list of elements
+            when the API are "paginated". (content list is divided in more
+            than one page)
+        """
```

Comment:
The double quotes are consistent with the other functions in this file. Personally I think a standard for either single or double quotes should be set for the entire project and should be changed in all cases. The changes were mostly accidental, when modifying/checking the code I ran my formatter on the code which converts to single quotes.